src/Viewer.js

```diff
@@ -136,9 +136,14 @@ function Viewer(domElement, opts) {
   setFullSize(this._controlContainer);
 
   // Prevent bounce scroll effect on iOS.
-  this._controlContainer.addEventListener('touchstart', function(event) {
-    event.preventDefault();
-  });
+  // Applied only for iPhone, as Android's events must have the default action to allow a proper hotspots' usage
+  var userAgent = navigator.userAgent || navigator.vendor || window.opera;
```

Comment:
We use the `bowser` library for user agent detection. Please do `var browser = require('bowser')` at the top, then rewrite this as `if (browser.ios)`.